packages/core/src/routes/index.ts

```diff
@@ -21,34 +15,9 @@ export class HttpServer {
   }
 
   private registerRoutes() {
```

Comment:
if everything is commented out, remove the registration on the bottom, don't leave it there. It's probably for the best to remove this entire file, it's not needed without these routes. Also removal of the "waitForExistence" that concerns the router, is required